docs/docs/configuration/overview.md

```diff
@@ -419,6 +420,20 @@ It is recommended to use `--session-store-type=redis` when expecting large sessi
 
 You have to substitute *name* with the actual cookie name you configured via --cookie-name parameter. If you don't set a custom cookie name the variable  should be "$upstream_cookie__oauth2_proxy_1" instead of "$upstream_cookie_name_1" and the new cookie-name should be "_oauth2_proxy_1=" instead of "name_1=".
 
+You can use the `--auth-endpoint-accept-unauthenticated=true` option to allow requests to propagate without logging in, whilst still passing the Authorization header when applicable:
+```yaml
```

Comment:
I was debating on stating that the `nginx.ingress.kubernetes.io/auth-signin` annotation could be left out. That would probably be better and more concise, but I am unsure.

Reply:
I think this would be good to include please